bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_10/A5_10_18.java

```diff
@@ -14,15 +14,30 @@
 
 import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.openhab.binding.enocean.internal.messages.ERP1Message;
+import org.openhab.core.library.types.DecimalType;
+import org.openhab.core.types.State;
 
 /**
  *
  * @author Daniel Weber - Initial contribution
+ * 
+ *         From A5_10_18 up to A5_10_17 temperature is given as a 8Bit value (range: 250(!)..0).
+ *         Therefore higher values mean lower temperatures.
+ *         Temperature range 0..40.
```

Comment:
I can't make suggestions to existing code, but can you reformat this to:
```java
/**
 * From A5_10_18 up to A5_10_17 temperature is given as a 8Bit value (range: 250(!)..0).
 * Therefore higher values mean lower temperatures.
 * Temperature range 0..40.
 *
 * @author Daniel Weber - Initial contribution
 */
```

This will ensure it will be correctly shown in Javadoc and IDE's.